scripts/consensus.py

```diff
@@ -33,14 +38,171 @@ def concatenate_data(profile_list: list) -> pd.DataFrame:
         concat_df.columns.str.startswith("Metadata")
     ]
     concat_metadata_df = concat_df.loc[:, concat_metadata_cols]
+
     concat_df = concat_df.drop(concat_metadata_cols, axis="columns")
-    concat_df = pd.concat([concat_metadata_df, concat_df])
+    concat_df = pd.concat([concat_metadata_df, concat_df], axis="columns")
 
     # dropping columns with na values
     na_cols = get_na_columns(concat_df, cutoff=0)
     concat_df = concat_df.drop(na_cols, axis="columns")
 
-    return concat_df
+    # dropping Costes Features
+    costes_cols_to_drop = [x for x in concat_df.columns if "costes" in x.lower()]
+    concat_df = concat_df.drop(costes_cols_to_drop, axis="columns")
+
+    x_groupby_cols = ["Metadata_gene_name", "Metadata_pert_name", "Metadata_cell_line"]
+
+    x_metacount_df = (
+        concat_df.loc[:, x_groupby_cols]
+        .assign(n_measurements=1)
+        .groupby(x_groupby_cols)
+        .count()
+        .reset_index()
+        .assign(data_type="cell_painting")
```

Comment:
i don't think we need this

Reply:
has been removed in: https://github.com/WayScience/CytoPipe/pull/11/commits/2f55798431b9dcd229f6193810b74a40c8211d7b